beacon-chain/forkchoice/doubly-linked-tree/forkchoice_test.go

```diff
@@ -36,7 +36,7 @@ func prepareForkchoiceState(
 		ParentRoot: parentRoot[:],
 	}
 
-	executionHeader := &enginev1.ExecutionPayloadHeader{
```

Comment:
With these changes, are we completely removing the existing test conditions and always default to epbs?

Reply:
I thought this branch is specific for ePBS but I probably overlooked the purpose of this function. How have been Prysm conducted fork choice test over past upgrades?